bundles/http/Citrix/SAPHana/script.js

```diff
@@ -0,0 +1,876 @@
+        PersonWorkAgreement: timeEntryObj.personWorkAgreement,
+        TimeSheetDate: `/Date(${timeEntryObj.date}​)/`,
+        TimeSheetStatus: timeEntryObj.timeSheetStatus,
+        TimeSheetOperation: timeEntryObj.timeSheetOperation
+    }
+    if (timeEntryObj.timeEntryId) {
+        const startedAtMoment = moment(timeEntryObj.startedAt)
+        const minutesDiff = timeEntryObj.endAt.diff(startedAtMoment, 'minutes');
+        const total = minutesDiff / 60
+        body.TimeSheetRecord = timeEntryObj.timeEntryId
+        body.TimeSheetDataFields.RecordedHours = total.toString()
+        body.TimeSheetDataFields.RecordedQuantity = total.toString()
+
+    }
+
+    const respPOST = await client.fetch('API_MANAGE_WORKFORCE_TIMESHEET/TimeSheetEntryCollection', {
+        headers: headers,
+        method: 'POST',
+        body: JSON.stringify(body)
+    });
+
+    if (!respPOST.ok) {
+        throw new Error(`[respPOST] ## Error Response -> (${JSON.stringify(respPOST)}​)`);
+    }
+
+    return respPOST.json()
+}
+
+async function getCreatedTimeEntryApiRequest(client, personWorkAgreementExternalId, companyCode, timeSheetRecord) {
+
+    const timeEntryUrl = `API_MANAGE_WORKFORCE_TIMESHEET/TimeSheetEntryCollection(PersonWorkAgreementExternalID='${personWorkAgreementExternalId}​',CompanyCode='${companyCode}​',TimeSheetRecord='${timeSheetRecord}​')`
+
+    const getCreatedTimeEntry = await client.fetch(timeEntryUrl, defaultGetOptions);
+
+    if (!getCreatedTimeEntry.ok) {
+        throw new Error(`Could not retrieve data from time entry (${getCreatedTimeEntry.status}​: ${getCreatedTimeEntry.statusText}​)`);
+    }
+
+    const timeEntry = await getCreatedTimeEntry.json()
+
+    return timeEntry.d
+}
+
+async function getEmployeeBusinessPartner(client, user_email) {
+
+    const employeeDataUrl = `YY1_EMPLOYEEDATA_CDS/YY1_EmployeeData?$filter=DefaultEmailAddress eq '${user_email}​'`
+
+    const employeeDataGET = await client.fetch(employeeDataUrl, defaultGetOptions);
+
+    if (!employeeDataGET.ok) {
+        throw new Error(`Could not Retrieve Employee Data(${employeeDataGET.status}​: ${employeeDataGET.statusText}​)`);
+    }
+
+    const employeeDataResponse = await employeeDataGET.json()
+
+    return employeeDataResponse.d.results[0].BusinessPartner
+}
+
+async function getWorkAgreement(client, employeeBusinessPartner) {
+    const workAgreementUrl = `YY1_WORKAGREEMENTDETAILS_CDS/YY1_WorkAgreementDetails?$filter=Person eq '${employeeBusinessPartner}​'`
+
+    const workAgreementGET = await client.fetch(workAgreementUrl, defaultGetOptions);
+
+    if (!workAgreementGET.ok) {
+        throw new Error(`Could not Retrieve Employee Data(${workAgreementGET.status}​: ${workAgreementGET.statusText}​)`);
+    }
+
+    const workAgreementResponse = await workAgreementGET.json()
+
+    return workAgreementResponse.d.results[0]
+}
+async function getTimeEntries({ dataStore, client }, incrementalSync) {
+    let timeEntries = []
```

Comment:
```suggestion
    const timeEntries = []
```

Reply:
unused variable removed, unknown characters that caused eslint alert removed.